arangod/RestHandler/RestShutdownHandler.cpp

```diff
@@ -81,7 +81,7 @@ RestStatus RestShutdownHandler::execute() {
     builder.add(VPackValue(true));
     AgencyCommResult result = agency.setValue("Shutdown", builder.slice(), 0.0);
     if (!result.successful()) {
-      generateError(rest::ResponseCode::SERVER_ERROR, 500);
+      generateError(rest::ResponseCode::SERVER_ERROR, TRI_ERROR_HTTP_SERVER_ERROR);
```

Comment:
Can we turn `result` into a more useful error here, e.g. by leveraging `result.asResult()`?

Reply:
Let's do this in a different PR.